Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -272,9 +272,9 @@ private async Task RegisterServerEndpoints()
                 {
                     CacheLifetime = TimeSpan.FromSeconds(1800), //How long SSDP clients can cache this info.
                     Location = uri, // Must point to the URL that serves your devices UPnP description document. 
-                    FriendlyName = "Emby Server",
-                    Manufacturer = "Emby",
-                    ModelName = "Emby Server",
+                    FriendlyName = "Jellyfin",
+                    Manufacturer = "Jellyfin",
+                    ModelName = "Jellyfin Media Player",
```

Comment:
I'd say it should be "Jellyfin Media Server", not player.

Reply:
Media is redundant, Jellyfin Server would be better.

Reply:
Agreed.

Reply:
Changed